app/tut/ga/creatlach.md

```diff
@@ -0,0 +1,967 @@
+mar a bhíonn ainmneacha bainte le [[athróga|variables]].
+
+Mar shampla, ba féidir linn oibiacht a bheith againn a [[léiríonn duine éigin|represents a person]]:
+B'fhéidir ba iad [[`aois`|age]], [[`seoladh`|address]] nó `ainm` a bhall agus
+[[stórálfadh|it would store]] siad aois an duine, seoladh an duine agus ainm an duine.
+
+Is féidir iompraíochtaí speisialta a bheith ag oibiachtaí freisin, ach feicimid níos mó faoi sin
+níos déanaí ar an leathanach seo.
+
+Conas a úsáidimid oibiachtaí?, ba chóir dúinn an siombail `@` a úsáid!
+
+## Cad é `@`?
+
+Nuair ba mhaith linn ball oibiachta a roghnú, bainimid úsáid as an tsiombail `@` agus ainm an bhaill
+mar seo:
+
+```
+<ainm an bhaill>@<an oibiacht>
+```
+
+[[Samhlaigh|Imagine]] anois go bhfuil oibiacht againn a léiríonn [[crann|tree]] éigin san athróg
+`crann`, agus tá ball aige [[`airde`|height]] a stórálann airde an chrainn. Scríobhfaimis
+`airde@crann`{.setanta} chun airde an chrainn a fháil.
+
+Nuair a scríobhaimid `ciorcal@stáitse`{.setanta} táimid ag iarr ó *Setanta* "Faigh luach an bhaill
+`ciorcal` ón oibiacht `stáitse`{.setanta}.". Dá bhrí sin nuair a scríobhaimid
+`ciorcal@stáitse(x, y, ga)` táimid ag roghnú an gníomh `ciorcal` ón oibiacht `stáitse`{.setanta}
+agus ag glaoch air (le `x`, `y` agus `ga` mar athróga).
+
+Is féidir linn an [[slonn|expression]] `ciorcal@stáitse(200, 200, 100)`{.setanta} a
+[[scaradh|separate]] ina dhá [[chéim|step]] chun an [[próiseas|process]] seo a thuiscint níos fearr.
+Ar dtús bainfimid úsáid as `@` chun an gníomh `ciorcal` a roghnú, tar éis é sin glaofaimid air.
+
+Cuirimis an gníomh `ciorcal` in athróg mar seo:
+
+```{.setanta .numberLines}
+gníomh_ciorcal := ciorcal@stáitse
+```
+
+Anois is féidir linn glaoch air:
+
+```{.setanta .numberLines}
+gníomh_ciorcal := ciorcal@stáitse
+gníomh_ciorcal(200, 200, 100)
+```
+
+Bain triail as anseo:
+
+{{{s
+gníomh_ciorcal := ciorcal@stáitse
+gníomh_ciorcal(200, 200, 100)
+}}}
+
+## [[Cruthú|Creation]]
+
+Chun oibiacht a chruthú tá rud uainn [[ar a dtugtar|called]] **"[[creatlach|outline]]"**. Is féidir
+linn creatlacha a úsáid chun oibiachtaí nua a dhéanamh. Is féidir creatlach amháin a úsáid chun a
+lán oibiachtaí a dhéanamh.
+
+Mar shampla is féidir linn creatlach a chruthú a [[sheasann do dhuine|represents a person]], agus
+ansin is féidir linn é a úsáid chun grúpa daoine a chur le chéile ón gcreatlach sin.
+
+# Creatlacha
+
+Cad é creatlach? Is [[cur síos|description]] é ar [[struchtúr oibiachta|structure of an object]].
+Nuair a cruthaímid creatlach, [[sainímid|we define]] ainm an creatlach agus liosta gníomhartha.
+Sainíonn an liosta gníomhartha seo iompraíocht na hoibiachta.
+
+Úsáidimid an [[eochairfhocail|keyword]] `creatlach`{.setanta} chun creatlach a chruthú mar seo:
+
+```{.setanta .numberLines}
+creatlach <ainm na hoibiachta> {
+    <liosta gníomhartha>
+}
+```
+
+Mar shampla: Seo creatlach [[`Simplí`|Simple]] a shainíonn oibiacht gan aon iompraíocht speisialta.
```

Comment:
shimplí

Reply:
Sa chás seo, is é "Simplí" ainm na creatlaí, ní aidiacht é